packages/trader/src/sass/app/modules/trading.scss

```diff
@@ -317,6 +317,68 @@
             right: 39%;
         }
     }
+    &__barriers-table {
+        position: absolute;
+        top: 0;
+        left: 0;
+        z-index: 100;
+        width: 100%;
+        height: calc(100vh - 4.8rem - 3.6rem - 1.6rem - 1.6rem);
```

Comment:
This values (`100vh - 4.8rem - 3.6rem - 1.6rem - 1.6rem)` were taken from existing scss code (Recent positions, right sidebar).